db/controller.js

```diff
@@ -0,0 +1,5 @@
+const model = require("./model")
+
+module.exports.getApi = function(req, res, next) {
+    res.status(200).send({endpoints: model.endpoints})
```

Comment:
is there a benefit of requiring the endpoints into another file and just exporting it when we could require the file directly into here? Models are generally for reading/requesting the data when we need to interact with a database for example whereas we don't need to do that here so we don't really need a controller! :)